Week2/index9.js

```diff
@@ -0,0 +1,32 @@
+//9
+x = 7
```

Comment:
I'm not sure if this will work without `const` or `let`, have you tested this one out?

Reply:
Yes, I've tested. It is working. I declared to x in index3.js